oe_theme.theme

```diff
@@ -1632,3 +1631,14 @@ function oe_theme_preprocess_field__media__oe_media_iframe__video_iframe__oe_the
   // Set video aspect ratio when rendering "oe_theme_main_content" display mode.
   $variables['ratio'] = $variables['element']['#object']->get('oe_media_iframe_ratio')->value;
 }
+
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ *
+ * Add block region as a suggestion.
+ */
+function oe_theme_theme_suggestions_block_alter(array &$suggestions, array $variables) {
+  /** @var \Drupal\block\Entity\Block $block */
+  $block = \Drupal::entityTypeManager()->getStorage('block')->load($variables['elements']['#id']);
```

Comment:
Don't we have the block object in $variables already?

Reply:
Sadly no, the configuration of the block is there but the region information is nowhere to be found.